src/ports/catalog/queries.ts

```diff
@@ -0,0 +1,355 @@
+    ? SQL`items.item_type = '`.append(FragmentItemType.EMOTE_V1).append(SQL`'`)
+    : SQL``
+}
+
+export const getWearableCategoryWhere = (
+  wearableCategory: WearableCategory
+) => {
+  return WearableCategory.validate(wearableCategory)
+    ? SQL`metadata_wearable.category = '`
+        .append(wearableCategory)
+        .append(SQL`'`)
+    : SQL``
+}
+
+export const getEmoteCategoryWhere = (emoteCategory: EmoteCategory) => {
+  return EmoteCategory.validate(emoteCategory)
+    ? SQL`metadata_emote.category = '`.append(emoteCategory).append(SQL`'`)
+    : SQL``
+}
+
+export const getEmotePlayModeWhere = (
+  emotePlayMode: EmotePlayMode | EmotePlayMode[]
+) => {
+  return Array.isArray(emotePlayMode)
+    ? emotePlayMode.length === 1
+      ? SQL`metadata_emote.loop = ${emotePlayMode[0] === EmotePlayMode.LOOP}`
+      : SQL``
+    : SQL`metadata_emote.loop = ${emotePlayMode === EmotePlayMode.LOOP}`
+}
+
+export const getSearchWhere = (search: string) => {
+  return SQL`items.search_text ILIKE '%' || ${search} || '%'`
+}
+
+export const getIsSoldOutWhere = () => {
+  return SQL`items.available = 0`
+}
+
+export const getIsOnSale = () => {
+  return SQL`((search_is_store_minter = true AND available > 0) OR listings_count > 0)`
+}
+
+export const getisWearableHeadAccessoryWhere = () => {
+  return SQL`items.search_is_wearable_head = true`
+}
+
+export const getWearableAccessoryWhere = () => {
+  return SQL`items.search_is_wearable_accessory = true`
+}
+
+export const getWearableGenderWhere = (
+  genders: (WearableGender | GenderFilterOption)[]
+) => {
+  const parsedGenders = []
+  if (genders.includes(GenderFilterOption.FEMALE)) {
+    parsedGenders.push('BaseFemale')
+  }
+  if (genders.includes(GenderFilterOption.MALE)) {
+    parsedGenders.push('BaseMale')
+  }
+  return SQL`items.search_wearable_body_shapes @> (${parsedGenders})`
+}
+
+export const getCreatorWhere = (creator: string | string[]) => {
+  return Array.isArray(creator)
+    ? SQL`items.creator = ANY(${creator})`
+    : SQL`items.creator = ${creator}`
+}
+
+export const getRaritiesWhere = (rarities: string[]) => {
+  return SQL`items.rarity = ANY(${rarities})`
+}
+
+export const getMinPriceWhere = (minPrice: string) => {
+  return SQL`(min_price >= ${minPrice} OR (price >= ${minPrice} AND available > 0))`
+}
+
+export const getMaxPriceWhere = (maxPrice: string) => {
+  return SQL`(max_price <= ${maxPrice} OR (price <= ${maxPrice} AND available > 0))`
+}
+
+export const getContractAddressWhere = (contractAddresses: string[]) => {
+  return SQL`items.collection = ANY(${contractAddresses})`
+}
+
+export const getOnlyListingsWhere = () => {
+  return SQL`(items.search_is_store_minter = false OR (items.search_is_store_minter = true AND available = 0)) AND listings_count > 1`
+}
+
+export const getOnlyMintingWhere = () => {
+  return SQL`items.search_is_store_minter = true AND available > 0`
+}
+
+export const getCollectionsQueryWheres = (filters: CatalogFilters) => {
```

Comment:
Is `getCollectionsQueryWheres` a typo?

```suggestion
export const getCollectionsQueryWhere = (filters: CatalogFilters) => {
```

Reply:
I was intentional :p but `where` sounds a little bit better